.github/workflows/pants.yaml

```diff
@@ -24,6 +24,6 @@ jobs:
     - name: bootstrap
       run: ./pants --version
     - name: Lint
-      run: ./pants --changed-since=origin/master lint
+      run: ./pants lint --changed-since=SHA/reference/branch
```

Comment:
Is this some github actions placeholder?

Reply:
no. I misunderstood @Eric-Arellano's comment in chat.
I'll follow up on this later today.